### PR TITLE
Adds support for multiple inputs in EmbeddingsRequest

### PR DESCRIPTION
**Pull request by ZeekoZhu**
- Adds support for multiple inputs in `EmbeddingsRequest`

### CODE REVIEW

**Review by StephenHodgson — DISMISSED** [review]
This looks great! Could we also add unit tests and bump the version and add change log?

Thanks

**Comment by ZeekoZhu** [comment]
@StephenHodgson done, please review

File: OpenAI-DotNet-Tests/TestFixture_05_Embeddings.cs [hunk 1]
```diff
@@ -14,5 +14,16 @@ public async Task Test_1_CreateEmbedding()
...
             Assert.IsNotEmpty(result.Data);
         }
+
+        [Test]
+        public async Task Test_2_CreateEmbeddingsWithMultipleInputs()
+        {
+            var api = new OpenAIClient(OpenAIAuthentication.LoadFromEnv());
+            Assert.IsNotNull(api.EmbeddingsEndpoint);
+            var result = await api.EmbeddingsEndpoint.CreateEmbeddingAsync(
+                new[] { "The food was delicious and the waiter...", "The food was terrible and the waiter..." });
```
Comment (StephenHodgson) [inline OpenAI-DotNet-Tests/TestFixture_05_Embeddings.cs:24]: let's either break this out into a local variable or make it one line.

File: OpenAI-DotNet/Embeddings/EmbeddingsEndpoint.cs [hunk 2]
```diff
@@ -37,6 +38,28 @@ protected override string GetEndpoint()
...
+        /// Defaults to: text-embedding-ada-002
+        /// </param>
+        /// <param name="user">
+        /// A unique identifier representing your end-user, which can help OpenAI to monitor and detect abuse.
+        /// </param>
+        /// <returns><see cref="EmbeddingsResponse"/></returns>
+        public async Task<EmbeddingsResponse> CreateEmbeddingAsync(
+            IList<string> input,
+            Model model = null,
+            string user = null)
```
Comment (StephenHodgson) [inline OpenAI-DotNet/Embeddings/EmbeddingsEndpoint.cs:60]: Please make all one line for parameters

File: OpenAI-DotNet/Embeddings/EmbeddingsRequest.cs [hunk 3]
```diff
@@ -29,13 +30,42 @@ public EmbeddingsRequest(string input, Model model = null, string user = null)
                 throw new ArgumentNullException(nameof(input), $"Missing required {nameof(input)} parameter");
             }
 
-            Input = input;
+            Input = new List<string> { input };
+            Model = model ?? new Model("text-embedding-ada-002");
+            User = user;
+        }
+
+
```
Comment (StephenHodgson) [inline OpenAI-DotNet/Embeddings/EmbeddingsRequest.cs:38]: not an extra space here

File: OpenAI-DotNet/Embeddings/EmbeddingsRequest.cs [hunk 4]
```diff
@@ -29,13 +30,42 @@ public EmbeddingsRequest(string input, Model model = null, string user = null)
...
+        /// </param>
+        /// <param name="model">
+        /// The <see cref="OpenAI.Models.Model"/> to use.
+        /// Defaults to: text-embedding-ada-002
+        /// </param>
+        /// <param name="user">
+        /// A unique identifier representing your end-user, which can help OpenAI to monitor and detect abuse.
+        /// </param>
+        /// <exception cref="ArgumentNullException">A valid <see cref="input"/> string is a Required parameter.</exception>
+        public EmbeddingsRequest(IList<string> input, Model model = null, string user = null)
```
Comment (StephenHodgson) [inline OpenAI-DotNet/Embeddings/EmbeddingsRequest.cs:55]:
```suggestion
        public EmbeddingsRequest(List<string> input, Model model = null, string user = null)
```

File: OpenAI-DotNet/Embeddings/EmbeddingsRequest.cs [hunk 5]
```diff
@@ -29,13 +30,42 @@ public EmbeddingsRequest(string input, Model model = null, string user = null)
...
+            }
+
+            Input = new List<string>(input);
             Model = model ?? new Model("text-embedding-ada-002");
             User = user;
         }
 
         [JsonPropertyName("input")]
-        public string Input { get; }
+        public IList<string> Input { get; }
```
Comment (StephenHodgson) [inline OpenAI-DotNet/Embeddings/EmbeddingsRequest.cs:68]:
```suggestion
        public List<string> Input { get; }
```

File: OpenAI-DotNet/Embeddings/EmbeddingsRequest.cs [hunk 6]
```diff
@@ -29,13 +30,42 @@ public EmbeddingsRequest(string input, Model model = null, string user = null)
...
+        /// The <see cref="OpenAI.Models.Model"/> to use.
+        /// Defaults to: text-embedding-ada-002
+        /// </param>
+        /// <param name="user">
+        /// A unique identifier representing your end-user, which can help OpenAI to monitor and detect abuse.
+        /// </param>
+        /// <exception cref="ArgumentNullException">A valid <see cref="input"/> string is a Required parameter.</exception>
+        public EmbeddingsRequest(IList<string> input, Model model = null, string user = null)
+        {
+            if (input.Count == 0)
```
Comment (StephenHodgson) [inline OpenAI-DotNet/Embeddings/EmbeddingsRequest.cs:57]: We should also check if it's null, just in case

**Review by StephenHodgson — CHANGES_REQUESTED** [review]
See comments

**Pull request merged**